### ISSUE TITLE
TARGET_IS_CLICKABLE is not working. Neither in the sample app.

### ISSUE THREAD

**Issue by kun-yu-tsai** [Nescafemix/hintcase#13]
First, I have to say you guys are just amazing. This library is just AWESOME. Our whole team love it very much!

After I implemented it, I found the "TARGET_IS_CLICKABLE" flag doesn't work. I tried test on the Sample App and the result was the same. In Target Example 3, the Radio button couldn't be switch even though the flag is TARGET_IS_CLICKABLE.

I looked into the code, I believe it's something happend in dispatchEvent. Maybe it wasn't rekon as a "Click Event"? Anyone has any idea? Thanks!

**Comment by kun-yu-tsai** [comment]
I have fixed it and I'll post my solution here.

Inside the **HintCaseView.java** file. Make the change as below:

```
    /**
     * As the touch_down happened in target shape, this will become true.
     * Used for the decision shall we invoke click event for target view.
     */
    private boolean mPressedInShape;

@Override
public boolean onTouchEvent(MotionEvent event) {

        boolean consumeTouchEvent = true;

        switch(event.getAction()){
            case MotionEvent.ACTION_DOWN:
                if (closeOnTouch) performHide();
                mPressedInShape = shape.isTouchEventInsideTheHint(event);
                break;
            case MotionEvent.ACTION_MOVE:
                if(!shape.isTouchEventInsideTheHint(event))
                    mPressedInShape = false;
                break;
            case MotionEvent.ACTION_UP:
                if(mPressedInShape && targetView != null && isTargetClickable && shape.isTouchEventInsideTheHint(event)){
                    targetView.performClick();
                }
                break;
        }

        return consumeTouchEvent;
    }
```

And it looks like I can't send out a pull-request because it's not allowed, or is it just I didn't figure out how to do it? 

Anyway, cheers man!

**Comment by qbait** [comment]
Joan, are you going to repair it? I'm using gradle to download dependency, so would be great to have this fix in the library.

**Comment by Nescafemix** [comment]
Fist of all, I'm sorry I took so long to respond (I was on parental leave and vacation).
First of all, @colza Thanks a lot!!! I'm going to implement your solution as soon as possible.

@qbait, yes, you can rely on it! I hope to do it in next days.

THX AGAIN!!

**Comment by qbait** [comment]
Hi @Nescafemix, any update on it?

**Comment by Nescafemix** [comment]
@qbait I'm working on it. I'm going to try to publish today the 1.0.3 version with this fix. :)

**Comment by Nescafemix** [comment]
#17 fixes this bug!.
The problem will be fixed in 1.0.3!

**Comment by Nescafemix** [comment]
@colza, @qbait , the version 1.0.3 is published. Thanks for your help!
